tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNet/HttpConfigurationStruct.cs

```diff
@@ -0,0 +1,27 @@
+// <copyright file="HttpConfigurationStruct.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections;
+using System.ComponentModel;
+using System.Reflection;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNet
+{
+    /// <summary>
+    /// System.Web.Http.HttpConfiguration
+    /// </summary>
+    [DuckCopy]
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct HttpConfigurationStruct
+    {
+        /// <summary>
+        /// Gets the MessageHandlers collection
+        /// </summary>
+        [Duck(BindingFlags = DuckAttribute.DefaultFlags | BindingFlags.IgnoreCase)]
```

Comment:
`DuckAttribute.DefaultFlags` doesn't hit the same problem you found with `DuckField` in #1601 does it?

Reply:
No it doesn't hit the same issue because `BindingFlags` is defined in corlib, not in a custom assembly